src/NServiceBus.RabbitMQ/Receiving/MessageConverter.cs

```diff
@@ -33,36 +33,44 @@ public string RetrieveMessageId(BasicDeliverEventArgs message, Dictionary<string
         public Dictionary<string, string> RetrieveHeaders(BasicDeliverEventArgs message)
         {
             var properties = message.BasicProperties;
+            var headers = properties.Headers;
```

Comment:
The existence of both `headers` and `messageHeaders` hurts readability a little. I'd probably prefer if this variable didn't exist and the deserialized headers variable remained named `headers`.